src/chrome/content/rules/bghm.de.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+	This domain contains a wildcard DNS and a wildcard certificate.
+
+	Not found:
+		entwicklung.extranet.bghm.de
+		rollin.extranet.bghm.de
+		www.receiver.bghm.de
+-->
+<ruleset name="BGHM.de">
+
+	<target host="bghm.de" />
+	<target host="*.bghm.de" />
```

Comment:
It's bad to have so many exclusions. I would prefer a list of valid targets to simplify this rule.